### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Useaconnectionstring_AttachwithConnectionString_Screenreader : Voice Over unnecessarily announcing display name Four times.

### ISSUE THREAD

**Issue by Suraj1200499** [microsoft/AzureStorageExplorer#2312]
GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1; #VoiceOver; #Mac;  

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.11.0
MAC Version: Catalina 10.15 

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect Dialog" pane present on left side of screen and hit enter.
3. Select "Use a connection string" radio button and hit enter on Next button.
4. "Attach with Connection string" screen gets opened
5. Start Voice Over and navigate using the tab key to display name field.
6. Observe the Voice Over announcement.

**Actual:**
VoiceOver announces the 'Display name:' field, label four times.
For eg: VoiceOver says, "Display name: Display name: , Display name: Display name:, edit text"

**Expected:**
When VoiceOver users navigate to the 'Attach with Connection String' dialog,  the label of all fields should be announced once.
For eg: "Display name:, edit text"

**UserImapct:**
When VoiceOver users navigate to 'Attach with Connection String' dialog, VoiceOver announces the 'Display name:' field label four times which may confuse the screen reader user about the right field name and its purpose.

**Comment by MRayermannMSFT** [comment]
Same root cause as https://github.com/microsoft/AzureStorageExplorer/issues/1904 ,which I already checked in a fix for.

**Comment by Suraj1200499** [comment]
@MRayermannMSFT 
Bug#2312[https://github.com/microsoft/AzureStorageExplorer/issues/2312](url) and Bug#1904[https://github.com/microsoft/AzureStorageExplorer/issues/1904](url) are logged for two different Platforms(Mac and Win32).
Issue#2312 is logged for Mac(VoiceOver) and Issue#1904 is logged for Win32(NVDA/JAWS).

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by Suraj1200499** [comment]
@MRayermannMSFT Verified this issue in Microsoft Azure Tools Storage Explorer Application Version: 1.12.0, this issue is no longer reproduced. Hence Closing the bug.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Mac-Sep2019;
